tests/test_icon_outputs.py

```
@@ -65,12 +76,49 @@ def _ico_frame(size):
+    so it runs on whatever CI job has Pillow installed, same as the rest
+    of this module's Pillow-gated tests.
+    """
+    Image = _pil_image()
+    with Image.open(ICNS) as im:
+        return im.icns.getimage((size, size, scale)).convert("RGBA").copy()
+
+
+def _abs_diff(a: PILImage, b: PILImage) -> int:
     """Sum of absolute channel differences. Uses tobytes() rather than
     getdata(), which Pillow deprecates and removes in 14."""
     return sum(abs(x - y) for x, y in zip(a.tobytes(), b.tobytes()))
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)